images/vli/sle15_sp1/config.kiwi

```diff
@@ -23,7 +23,7 @@
         <timezone>UTC</timezone>
     </preferences>
     <preferences profiles="Production">
-        <type image="oem" filesystem="xfs" bootloader="grub2" bootloader_console="console" initrd_system="dracut" kernelcmdline="splash=verbose loglevel=3 nomodeset net.ifnames=1 numa_balancing=disable transparent_hugepage=never intel_idle.max_cstate=1 processor.max_cstate=1 crashkernel=232M,high crashkernel=80M,low nohz=off skew_tick=1 intel_pstate=disable clocksource=tsc console=ttyS0,115200" firmware="uefi" bootpartition="false">
```

Comment:
Can we not add these for SLES12 SP4 VLI images ?

Reply:
sure, from the initial report it looked like you wanted this for sle15 only ? Could you be more specific which images needs it =
All of VLI or only a subset ? Thanks

Reply:
This applies to all of VLI.
Also, can you please refer to the question in init shell setting on the main issue ?
Thanks !!

Reply:
> This applies to all of VLI.

ok done. Thanks